pkg/image/validation/os.go

```diff
@@ -189,27 +206,24 @@ func validatePackages(os *image.OperatingSystem) []FailedValidation {
 		})
 	}
 
-	if len(os.Packages.AdditionalRepos) > 0 {
-		urlSlice := []string{}
-
-		for _, repo := range os.Packages.AdditionalRepos {
-			if repo.URL == "" {
-				msg := "Additional repository list contains an entry with empty 'url' field."
-				failures = append(failures, FailedValidation{
-					UserMessage: msg,
-				})
-			}
-
-			urlSlice = append(urlSlice, repo.URL)
-		}
+	var repoURLs []string
 
-		if duplicates := findDuplicates(urlSlice); len(duplicates) > 0 {
-			duplicateValues := strings.Join(duplicates, ", ")
-			msg := fmt.Sprintf("The 'additionalRepos' field contains duplicate repos: %s", duplicateValues)
+	for _, repo := range os.Packages.AdditionalRepos {
+		if repo.URL == "" {
 			failures = append(failures, FailedValidation{
-				UserMessage: msg,
+				UserMessage: "The 'url' field is required for all entries under 'additionalRepos'.",
```

Comment:
This is only reworded for consistency with all other functions within the module.